src/engine/source/kvdb/src/kvdb.cpp

```diff
@@ -671,13 +676,26 @@ bool KVDB::write(const std::string& key,
                  const std::string& value,
                  const std::string& columnName)
 {
+    Metrics::instance().addCounterValue("Kvdb.WritesCounter", 1UL);
     return mImpl->write(key, value, columnName);
 }
 
 std::variant<std::string, base::Error> KVDB::read(const std::string& key,
                                       const std::string& columnName)
 {
-    return mImpl->read(key, columnName);
+    Metrics::instance().addCounterValue("Kvdb.ReadsCounter", 1UL);
+
+    auto startTime = std::chrono::high_resolution_clock::now();
+
+    auto response = mImpl->read(key, columnName);
+
+    auto elapsedTime = std::chrono::high_resolution_clock::now() - startTime;
+    auto microseconds = std::chrono::duration_cast<std::chrono::microseconds>(elapsedTime).count();
+    auto result = static_cast<uint64_t>(microseconds);
+
+    Metrics::instance().addHistogramValue("Kvdb.AccessTimeDBHistogram", result);
```

Comment:
you could directly do  `Metrics::instance().addHistogramValue("Kvdb.AccessTimeDBHistogram", rstatic_cast<uint64_t>(microseconds););`